dlp/risk.py

```diff
@@ -49,8 +49,34 @@ def numerical_risk_analysis(project, table_project_id, dataset_id, table_id,
     # potentially long-running operations.
     import google.cloud.pubsub
 
-    # This sample also uses threading.Event() to wait for the job to finish.
-    import threading
+    def callback(message):
+        try:
+            if (message.attributes['DlpJobName'] == operation.name):
+                # This is the message we're looking for, so acknowledge it.
+                message.ack()
+
+                # Now that the job is done, fetch the results and print them.
+                job = dlp.get_dlp_job(operation.name)
+                results = job.risk_details.numerical_stats_result
+                print('Value Range: [{}, {}]'.format(
+                    results.min_value.integer_value,
+                    results.max_value.integer_value))
+                prev_value = None
+                for percent, result in enumerate(results.quantile_values):
+                    value = result.integer_value
+                    if prev_value != value:
+                        print('Value at {}% quantile: {}'.format(
+                              percent, value))
+                        prev_value = value
+                subscription.set_result(None)
+            else:
+                # This is not the message we're looking for.
+                message.drop()
+        except Exception as e:
+            # Because this is executing in a thread, an exception won't be
```

Comment:
Is this still running in a thread even with the new system, or is it running in the main thread? It's possible we no longer need this section.

Repeated below as well